requirements.txt

```diff
@@ -18,7 +18,7 @@ itsdangerous==2.0.1
 Jinja2==3.0.3
 jmespath==0.10.0
 joblib==1.1.0
-kubernetes==24.2.0
+kubernetes==25.3.0
```

Comment:
Why not 25.5?

Reply:
Python libraries are not released with the same numbers
https://pypi.org/project/kubernetes/#history